openmp/runtime/src/kmp_affinity.cpp

```diff
@@ -2971,6 +2973,9 @@ static bool __kmp_affinity_create_cpuinfo_map(int *line,
     *msg_id = kmp_i18n_str_CantRewindCpuinfo;
     return false;
   }
+#else // KMP_OS_AIX
+  unsigned num_records = __kmp_xproc;
+#endif
```

Comment:
I tend to prefer inverting the logic and having the AIX bits.

Reply:
Changed the code per suggestion, thanks!